pom.xml

```diff
@@ -18,33 +18,32 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>co.cask</groupId>
+  <groupId>io.cdap.plugin</groupId>
   <artifactId>topn</artifactId>
   <version>1.2.0-SNAPSHOT</version>
```

Comment:
do we need to bump the project version also?

Reply:
no, release/1.2 is a new branch i created. I'll bump the version of develop after i merge release to develop

Reply:
actually.. yes, might as well put this to 1.2.1-SNAPSHOT